File: 프로그래머스/unrated/181879. 길이에 따른 연산/길이에 따른 연산.java
import java.util.Arrays;
class Solution {
    public int solution(int[] num_list) {
        int length = num_list.length;
        return (length >= 11) ? Arrays.stream(num_list).sum() : Arrays.stream(num_list).reduce(1,(a,b) -> a * b);
    }
}

File: 프로그래머스/0/181879. 길이에 따른 연산/길이에 따른 연산.cpp
#include <string>
#include <vector>
#include <numeric>
using namespace std;

int solution(vector<int> num_list) {
    int n = num_list.size();
    int answer = 0;
    if(n <= 10) {
        answer = 1;
        for(int i = 0; i < n; i++){
            answer *= num_list[i];
        }
    }else{
       answer = accumulate(num_list.begin(), num_list.end(), answer);
    }
    return answer;
}